Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -264,9 +309,21 @@ You can install Cilium on any Kubernetes cluster. Pick one of the options below:
 
            cilium install
 
-    .. group-tab:: AKS
+    .. group-tab:: AKS (BYOCNI)
+
+       .. include:: requirements-aks-byocni.rst
+
+       **Install Cilium:**
+
+       Install Cilium into the AKS cluster:
+
+       .. code-block:: shell-session
+
+           cilium install --azure-resource-group "${AZURE_RESOURCE_GROUP}"
```

Comment:
should we have something else to set byocni flag as well ? I think this command is just for AKS ipam mode.

Reply:
As noted above: this is handled by the CLI. You can check out https://github.com/cilium/cilium-cli/pull/899 for the full details on the implementation :)

Reply:
oh right, do we still need to pass datapath-mode flag ? :thinking: